src/main/kotlin/com/sourcegraph/cody/ui/web/WebUIService.kt

```diff
@@ -122,6 +130,22 @@ class WebUIService(private val project: Project) {
     }
   }
 
+  private fun createWebUIProxy(delegate: WebUIHost): WebUIProxy? =
+      try {
+        missingJcefExceptionOccurred.set(false)
+        WebUIProxy.create(delegate)
+      } catch (e: IllegalStateException) {
+        if (e.message == JCEF_NOT_SUPPORTED_MESSAGE) {
```

Comment:
This is potentially fragile.
Maybe instead we can save the exception, and then when refreshing panel if jcef runtime is missing then render current notification, and if it is not missing but webview error happened then show the error? Otherwise we will be stuck with the spinner forever anyway, no?

Reply:
If it's something else than missing jcef exception we simply rethrow it. This behaviour dees not change. We only add a handling for that particular exception.